src/modules/auth/constants/sign-up.constants.ts

```diff
@@ -0,0 +1,21 @@
+import type {
+  SignUpStep,
+  SignUpUserData,
+} from '@/modules/auth/types/sign-up.types';
+
+export const SIGN_UP_STEPS: { value: SignUpStep; text: string }[] = [
```

Comment:
About this file. How about we discuss the implementation of using constants? Recently I watched some developers and they had a point that spreading such data through parts of your application complicates it, as you have a lot of files, that are actually used by only one component and clearly belong to only them. I don't like having them in the component itself either. Maybe discussion will help us find some new approach.